ComponentKit/Core/CKComponentInternal.h

```diff
@@ -65,7 +65,7 @@
 @property (nonatomic, weak) UIView *rootComponentMountedView;
 
 /** For internal use only; don't touch this. */
-@property (nonatomic, strong, readonly) id scopeFrameToken;
+@property (nonatomic, strong, readonly) NSNumber *scopeFrameToken;
```

Comment:
That's what I had originally, but it felt a bit off since component scope collision detection uses this token in a set. Keeping this `id` works, just feels slightly less safe.